packages/pnpm-plugin/pnpmfile.cjs

```diff
@@ -0,0 +1,34 @@
+// @ts-check
+module.exports = {
+  hooks: {
+    /** @param {import("@pnpm/types").PnpmSettings} config */
+    updateConfig(config) {
+      // @ts-expect-error Property isn't in PnpmSettings for some reason
+      config.publicHoistPattern ??= [];
+      // @ts-expect-error Property isn't in PnpmSettings for some reason
+      config.publicHoistPattern.push(
```

Comment:
Should our patterns be _prepended_ so consumers can define overrides with `!`?

https://pnpm.io/settings#publichoistpattern